DARPA-ASKEM/terarium
# Calibration Configuration inferred params fix

# Description
Fix inferred params list so that it is not empty when we save model configs from calibration
The issue was that we were not providing the reference to the model config for the mapping, so we would create them and our `POST` would return a correct result but any `GET` after would not be able to find this field

# To Test:
Run a calibration,
Save it as a model config for reuse
Open it in the model config drilldown and see that this section is not all 0
<img width="642" alt="image" src="https://github.com/user-attachments/assets/5915dc24-1937-477e-a901-e706127a3688">
